src/OpenSage.Tools.BigEditor/UI/MainForm.cs

```diff
@@ -253,53 +301,91 @@ private void DrawFilesList()
-                    {
-                        filePath = ImGuiUtility.TrimToNullByte(filePath);
+            ImGui.EndChild();
 
-                        using (var entryStream = entry.Open())
-                        {
-                            using (var fileStream = File.OpenWrite(filePath))
-                            {
-                                entryStream.CopyTo(fileStream);
-                            }
-                        }
+            ImGui.EndChild();
+        }
 
-                        ImGui.CloseCurrentPopup();
-                    }
+        private void DrawStatusPanel()
+        {
+            ImGui.BeginChild("statusbar", new Vector2(0, 30), true, 0);
+
+            ImGui.Text($"{_bigArchive.FilePath} | Version: {_bigArchive.Version} | Size: {ImGuiUtility.GetFormatedSize(_bigArchive.Size)} | Files: {_bigArchive.Entries.Count}");
+
+            ImGui.SameLine();
 
-                    ImGui.SetItemDefaultFocus();
+            if (_currentFileName.CompareTo("") != 0)
+            {
+                ImGui.Text($"| Selected file: {_currentFileName}");
+            }
 
-                    ImGui.SameLine();
+            ImGui.EndChild(); // end statusbar
+        }
 
-                    if (ImGui.Button("Cancel"))
+        private void Export(string path)
+        {
+            foreach (var entry in _files)
+            {
+                string filePath = $"{path}{Path.DirectorySeparatorChar}{Path.GetDirectoryName(entry.FullName.Replace('\\', Path.DirectorySeparatorChar))}";
+                try
+                {
+                    Directory.CreateDirectory(filePath);
+                }
+                catch (Exception e)
+                {
+                    logger.Error(e.Message);
+                    return;
+                }
+
+                try
+                {
+                    using (var entryStream = entry.Open())
                     {
-                        ImGui.CloseCurrentPopup();
+                        using (var fileStream = File.OpenWrite($"{filePath}{Path.DirectorySeparatorChar}{entry.Name}"))
+                        {
+                            entryStream.CopyTo(fileStream);
+                        }
                     }
-
-                    ImGui.EndPopup();
+                }
+                catch (Exception e)
+                {
+                    logger.Error(e.Message);
                 }
             }
+        }
 
-            ImGui.Columns(1, null, false);
+        private void ExportFile(BigArchiveEntry entry, string filePath)
+        {
+            using (var entryStream = entry.Open())
+            {
+                using (var fileStream = File.OpenWrite(filePath))
+                {
+                    entryStream.CopyTo(fileStream);
+                }
+            }
+        }
 
-            ImGui.EndChild();
+        private void Import(string directory)
+        {
+            logger.Info("Import files from: {0}", directory);
```

Comment:
Could you add a TODO comment here since this is not implemented yet?

Reply:
Added comment